crates/biome_service/src/file_handlers/json.rs

```diff
@@ -584,6 +584,9 @@ fn fix_all(params: FixAllParams) -> Result<FixFileResult, WorkspaceError> {
                             return ControlFlow::Break(action);
                         }
                     }
+                    FixFileMode::ApplySuppressions => {
+                        // Not implemented
```

Comment:
```suggestion
                        // TODO: implement once a JSON suppression action is available
```

I figured we should add a `TODO` because `JSONC` has comments, so once the analyzer supports a suppression action, we can implement it